modules/light-clients/08-wasm/testing/wasm_endpoint.go

```diff
@@ -29,8 +29,11 @@ func (endpoint *WasmEndpoint) CreateClient() error {
 	checksum, err := types.CreateChecksum(Code)
 	require.NoError(endpoint.Chain.TB, err)
 
-	clientState := types.NewClientState(contractClientState, checksum, clienttypes.NewHeight(0, 1))
-	consensusState := types.NewConsensusState(contractConsensusState)
+	wrappedClientStateBz := clienttypes.MustMarshalClientState(endpoint.Chain.App.AppCodec(), CreateMockWrappedClientState(clienttypes.NewHeight(1, 5)))
```

Comment:
```suggestion
	wrappedClientStateBz := clienttypes.MustMarshalClientState(endpoint.Chain.App.AppCodec(), MockWrappedClientState)
```

Would this work?

Reply:
I wanted to use a different mock client state with a different height than `MockWrappedClientState` because there are some tests where the client state is updated with `MockWrappedClientState` and wanted to verify that the height actually changes.